governance_artifacts/pilot/run_pilot_acceptance_gates.py

```
@@ -0,0 +1,282 @@
+
+
+def check_zk_relayer() -> tuple[bool, str]:
+    rc, out = _run(["bash", "run_relayer_pipeline.sh"], cwd=GA / "zk", timeout=300)
+    ok = rc == 0 and "relayer pipeline complete" in out
+    line = next((l.strip() for l in out.splitlines() if "compiles" in l), "")
+    return ok, line or (out.strip()[-120:])
+
+
+def check_containment_tlc() -> tuple[bool, str]:
+    jar = GA / "tla" / "tools" / "tla2tools.jar"
+    rc, out = _run(
```

Ruff (0.15.20)

[warning] Unpacked variable `rc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)